src/applications/vaos/tests/mocks/helpers.v2.js

```diff
@@ -224,3 +224,56 @@ export function mockV2FacilitiesFetch(ids, data, children = false) {
     { data },
   );
 }
+
+/**
+ * Mocks the api call that fetches a list of appointment slots for direct scheduling
+ *
+ * @export
+ * @param {Object} params
+ * @param {string} siteId The VistA site id where slots are from
+ * @param {string} typeOfCareId The type of care id of the slots being requested
+ * @param {string} preferredDate The preferred date chosen by the user, which determines the date range fetched,
+ *    if startDate and endDate are not provided
+ * @param {MomentDate} startDate The start date for the apppointment slots
+ * @param {MomentDate} endDate The end date for the apppointment slots
+ * @param {string} [length=20] The length of the appointment slots
+ * @param {string} clinicId The VistA clinic id the slots are in
+ * @param {Array<VARSlot>} slots The list of slots to return from the mock
+ */
+export function mockAppointmentSlotFetch({
+  facilityId,
+  preferredDate,
+  startDate,
+  endDate,
+  clinicId,
+}) {
+  const start = startDate || preferredDate.clone().startOf('month');
+  const end =
+    endDate ||
+    preferredDate
+      .clone()
+      .add(1, 'month')
+      .endOf('month');
+
+  setFetchJSONResponse(
+    global.fetch.withArgs(
+      `${
+        environment.API_URL
+      }/vaos/v2/locations/${facilityId}/clinics/${clinicId}/slots?` +
+        `&start_date=${start.format('YYYY-MM-DDTHH:mm:ssZ')}` +
```

Comment:
Should be able to remove the format string here (and below) and do `.format()` instead